svc-update/update/simpleUpdate.go

```diff
@@ -294,3 +303,52 @@ func sortTargetList(Targets []string) (map[string][]string, error) {
 	}
 	return returnList, nil
 }
+
+func expandAggregatesCollection(token string, links []string) ([]string, error) {
```

Comment:
expand means increase in size, it this method properly named?

Reply:
I think 'expand' is the correct name.  It aligns with the meaning/usage of $expand as a query parameter.

Reply:
Lets leave expand in the name of a method